docs/source/beta.rst

```diff
@@ -430,68 +463,74 @@ After you set up your credentials and sources, you can run a Quipucords scan to
 
 To run a scan, use the following steps:
 
-Run the scan by using the ``scan start`` command, specifying one or more sources for the ``sources`` option:
+Run the scan by using the ``scan start`` command, specifying one or more sources for the ``sources`` option::
 
-  ``# qpc scan start --sources source_name1 source_name2``
+  # qpc scan start --sources source_name1 source_name2
 
-For example, if you want to scan the network source ``mynetwork`` and the Satellite source ``mysatellite6``, you would enter the following command:
+For example, if you want to scan the network source ``mynetwork`` and the satellite source ``mysatellite6``, you would enter the following command::
 
-  ``# qpc scan start --sources mynetwork mysatellite6``
+  # qpc scan start --sources mynetwork mysatellite6
 
 Showing Scan Results for an Active Scan
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 When you run the ``scan start`` command, the output provides an identifier for that scan. You can show the scan results to follow the status of the scan by using the ``scan show`` command and specifying the provided identifier.
 
 **IMPORTANT:** The ``scan show`` command can show results only after the scan starts running. You can also use this command on a scan that is completed.
 
-For example, you could run the following scan as the first scan in your environment:
+For example, you could run the following scan as the first scan in your environment::
 
-  ``# qpc scan start --sources mynetwork mysatellite6``
+  # qpc scan start --sources mynetwork mysatellite6
 
-The output for the command shows the following information, with ``1`` listed as the scan identifier.
+The output for the command shows the following information, with ``1`` listed as the scan identifier::
 
-  ``Scan "1" started``
+  Scan "1" started
 
-To show the scan results to follow the status of that scan, you would enter the following command:
+To show the scan results to follow the status of that scan, you would enter the following command::
 
-  ``# qpc scan show --id 1``
+  # qpc scan show --id 1
 
 Listing Scan Results
 ^^^^^^^^^^^^^^^^^^^^
-In addition to showing the status of a single scan, you can also show a list of all scans that are in progress or are completed on the Quipucords server. To show this list of scans, you use the ``scan list`` command. The output of this command includes the scan identifier, the source or sources for that scan, and the current state of the scan.
+In addition to showing the status of a single scan, you can also show a list of all scans that are in progress or are completed on the Quipucords server. To show this list of scans, you use the ``scan list`` command. The output of this command includes the scan identifier, the source or sources for that scan, and the current state of the scan:.
```

Comment:
Ends with `the scan:.`, is that right?

Reply:
No, typo from adding the new sentence after that one. Should be fixed with a new commit/push.